src/main/java/mate/jdbc/service/DriverServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.model.Driver;
+
+public class DriverServiceImpl implements DriverService {
+
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Driver get(Long id) {
+        return driverDao.get(id);
```

Comment:
because of this driverDao.get(id) will return Optional<Driver> lets rewrite this method with catching Exception